.pre-commit-config.yaml

```diff
@@ -0,0 +1,46 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+    -   id: check-added-large-files
+
+-   repo: https://github.com/PyCQA/isort
+    rev: "5.10.1"
+    hooks:
+        - id: isort
+
+-   repo: https://github.com/asottile/pyupgrade
+    rev: "v3.2.2"
+    hooks:
+        - id: pyupgrade
+          args: [--py37-plus]
+-   repo: https://github.com/hadialqattan/pycln
+    rev: "v2.1.2"
+    hooks:
+      - id: pycln
+
+-   repo: https://github.com/psf/black
+    rev: "22.10.0"
+    hooks:
+      - id: black
+        args: [--target-version=py37]
+
+-   repo: https://github.com/PyCQA/flake8
+    rev: "5.0.4"
+    hooks:
+      - id: flake8
+ci:
+    autofix_commit_msg: |
+        [pre-commit.ci] auto fixes from pre-commit.com hooks
+
+        for more information, see https://pre-commit.ci
+    autofix_prs: true
+    autoupdate_branch: ''
+    autoupdate_commit_msg: '[pre-commit.ci] pre-commit autoupdate'
+    autoupdate_schedule: weekly
+    skip: []
+    submodules: false
```

Comment:
Can I suggest adding https://github.com/floatingpurr/sync_with_poetry/ so that versions in pre-commit stay sync with those in Poetry ?

Also, Bandit ?

Reply:
We're still not using Poetry: please contribute this 🙂